alibi/utils/frameworks.py

```diff
@@ -69,9 +70,9 @@ def infer_device(predictor, predictor_type: str, framework: str) -> Union[None,
     """
 
     if framework == 'tensorflow':
-        return  # type: ignore
+        return None
     if predictor_type == 'blackbox':
-        return
+        return None
```

Comment:
I disagree with this. `return` means `return None` in `python`, this is a language convention. If the typing library doesn't know about this, then this is a problem with it so I say we should stick with the `# type:  ignore` directive.

Reply:
Ok fair enough, I just thought this is cleaner/shorter than doing the extra type declaration.

Reply:
I still insist :)

Reply:
Actually, I take back my original assertion. Having researched a bit, we can see that your modification makes sense in this case, according to [this](https://stackoverflow.com/questions/15300550/return-return-none-and-no-return-at-all) source.